modules/social_features/social_group/modules/social_group_invite/config/update/social_group_invite_update_8007.yml

```diff
@@ -0,0 +1,16 @@
+message.template.invited_to_join_group:
```

Comment:
I see just hook update, but not actual change to the `message.template.invited_to_join_group` from `config/install` folder.

Reply:
fixed